pkg/detectors/openvpn/openvpn.go

```diff
@@ -0,0 +1,111 @@
+package openvpn
+
+import (
+	"context"
+	b64 "encoding/base64"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+
+	clientIdPat     = regexp.MustCompile(detectors.PrefixRegex([]string{"openvpn"}) + `\b([A-Za-z0-9-]{3,40}\.[A-Za-z0-9-]{3,40})\b`)
```

Comment:
question: Do you know if clientIDs can contain `-` in them. I created a couple and none of them happened to have the `-`. If they don't contain the `-` could we omit them from the first capture group? I'm guessing if the domain has a `-` it would get included in the second which is fine. If the first portion can also contain a `-` plz feel free to ignore this comment.

Reply:
nit: For vars and consts we should try to capitalize acronyms where possible.
Ex: `clientIdPat` -> `clientIDPat`

[style guide ref](https://google.github.io/styleguide/go/decisions#initialisms)

Reply:
@ahrav You're right!, let me remove `-` from `clientIDPat`